.travis.yml

```diff
@@ -83,7 +83,7 @@ jobs:
         - powershell "$CONDA_PATH; conda install --yes -c msys2 m2-base m2-make m2w64-toolchain libpython"
         - powershell "$CONDA_PATH; pip install tox"
       script:
-        - powershell "$CONDA_PATH; tox -e py3"
+        - powershell "$CONDA_PATH; tox -e py3_mingw"
```

Comment:
```suggestion
        # the default compiler on windows, msvc, does not understand our compiler flags
        - powershell "$CONDA_PATH; tox -e py3_mingw"
```

Reply:
There is more to this than compiler flags https://travis-ci.org/themperek/cocotb/builds/646876123
If anything something "msvc is not supported" maybe.

Reply:
Sounds good to me. Can you reference `gh-1372` in that comment?

Reply:
```suggestion
        # TODO[gh-1372]: the default compiler on windows, msvc, is not supported
        - powershell "$CONDA_PATH; tox -e py3_mingw"
```